CHANGELOG.rst

```diff
@@ -32,7 +32,7 @@ v0.8.0 (2019-09-17)
    (`#43 <https://github.com/treasure-data/pytd/pull/43>`__, `#44 <https://github.com/treasure-data/pytd/pull/44>`__)
 -  Disable ``type``, one of the Treasure Data-specific query parameters, because it is conflicted with the ``engine`` option.
    (`#45 <https://github.com/treasure-data/pytd/pull/45>`__)
--  Add `td-pyspark <https://pypi.org/project/td-pyspark/>`__ dependency for easily accessing to the `td-spark <https://support.treasuredata.com/hc/en-us/articles/360001487167-Apache-Spark-Driver-td-spark-FAQs>`__ functionalities.
+-  Add `td-pyspark <https://pypi.org/project/td-pyspark/>`__ dependency for easily accessing to the `td-spark <https://tddocs.atlassian.net/wiki/spaces/PD/pages/1082513/Apache+Spark+Driver+td-spark+FAQs>`__ functionalities.
```

Comment:
I wonder which is the right place to point td-spark, official doc's FAQ, or [API doc](https://treasure-data.github.io/td-spark/).

@xerial @shimamoto Can you guide us which is the right pointer for td-spark?

Reply:
Strong +1 on the official API doc; TD's doc site doesn't help that much. Replaced: 7bb739eb1853afd845d25eeeeaccbe858ac7f040

Reply:
TD's doc site will be updated soon, but API is doc is suited to such up-to-date information for OSS libraries.